### PR TITLE
initial mkdocs

### PR DESCRIPTION
**Pull request by pvgenuchten**
a suggestion for a setup using mkdocs, based on work of @justb4 at https://github.com/Geonovum/ogc-api-testbed/tree/main/docs

### CODE REVIEW

**Comment by justb4** [comment]
Thanks for this @pvgenuchten ! I like the Material look, but I'm prejudiced ;-). 

Suggestion, something I may do in another PR: in the original Geonovum project we auto-published the docs with GH Workflows to https://apitestdocs.geonovum.nl, see the [Action File](https://github.com/Geonovum/ogc-api-testbed/blob/main/.github/workflows/deploy.docs.yml). Think we can do the same here. Creating a custom subdomain like `ws.pygeoapi.io` and run from there. For authoring a local `mkdocs` is necessary, but IMHO not required for WS participants.

**Comment by justb4** [comment]
(Ok I now see you mention GH Workflows already, I overlooked, we're on the right track).

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Diving into pygeoapi
+
+A 4 hour workshop to get familiarised with the pygeoapi software. A python package providing standardised web api's for querying, vizualising and processing spatial datasets.
+
+In this workshop we'll discuss the following topics:
+
+- [Introduction to pyGeoAPI](intro/index.md)
```
Comment (francbartoli) [inline docs/index.md:7]: shouldn't be `pygeoapi` lowercased?

File: mkdocs.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+site_name: Diving into pygeoapi
+site_description: "pygeoapi is an OGC Reference Implementation supporting numerous OGC API specifications. This workshop will cover publishing geospatial data to the Web using pygeoapi in support of the suite of OGC API standards."
+site_author: GeoPython
+copyright: "&copy; 2021 GeoPython"
+site_url: https://GeoPython.org
```
Comment (francbartoli) [inline mkdocs.yml:5]: `geopython` lowercased for the domain?

**Review by francbartoli — CHANGES_REQUESTED** [review]

**Review by francbartoli — APPROVED** [review]

**Pull request merged**